php/blocks/block-coauthor-avatar/class-block-coauthor-avatar.php

```diff
@@ -0,0 +1,100 @@
+<?php
+/**
+ * Co-Author Avatar
+ * 
+ * @package Co-Authors Plus
+ */
+
+namespace CoAuthors\Blocks; 
+
+use WP_Block;
+/**
+ * Block CoAuthor Avatar
+ */
+class Block_CoAuthor_Avatar {
+
+	/**
+	 * Register Block
+	 */
+	public static function register_block() : void {
+		register_block_type(
+			realpath( __DIR__ . '/../../..' ) . '/build/blocks/block-coauthor-avatar',
+			array(
+				'render_callback' => array( __CLASS__, 'render_block' ),
+			)
+		);
+	}
+	/**
+	 * Render Block
+	 * 
+	 * @param array $attributes
+	 * @param string $content
+	 * @param WP_Block $block
+	 * @return string
+	 */
+	public static function render_block( array $attributes, string $content, WP_Block $block ) : string {
+
+		$author = $block->context['cap/author'] ?? array();
+
+		if ( empty( $author ) ) {
+			return '';
+		}
+
+		$avatar_urls = $author['avatar_urls'] ?? array();
+
+		if ( empty( $avatar_urls ) ) {
+			return '';
+		}
+		
+		$display_name = $author['display_name'] ?? '';
+		$link    = $author['link'] ?? '';
+		$is_link = '' !== $link && $attributes['isLink'] ?? false;
+		$rel = $attributes['rel'] ?? '';
+
+		$size    = $attributes['size'] ?? 24;
```

Comment:
In https://github.com/Automattic/Co-Authors-Plus/pull/997/commits/fea90ac5f87961f28063bb93f370b43b3d0ac9a0 I updated this to use the first size from the `$avatar_urls` array instead of a magic number.